docs/source/dataformat.rst

```diff
@@ -889,6 +1095,60 @@ For example::
       </verse>
     </lyrics>
 
+Line repeat
+^^^^^^^^^^^
+
+In some songs not only the verses but also the lines may be repeated. Repeated verse can be managed
+with the ``<verseOrder>`` tag::
+
+    <verseOrder>v1 v1 c v2 v2 c</verseOrder>
+
+Repeating lines can be described with an optional attribute for lines::
+
+    <lines repeat="2">O my Jesus.</lines>
+
+The value of this attribute should be an integer with a value of 2 or more.
+
+Lyrics projectors and processors can display the above example like this::
+
+    𝄆 O my Jesus. 𝄇×2
+
+Or simply::
+
+    O my Jesus.
+    O my Jesus.
+
+Instrumental parts
+^^^^^^^^^^^^^^^^^^
+
+In some songs there are parts without lyrics, instrumental sections or vamps.
+OpenLyrics supports describing these parts, very similar to ``<verse>`` tags::
+
+    <lyrics>
+      <instrument name="i">
+        <lines>
+          <beat><chord root="B" structure="m3-5" /><chord root="A" bass="C#" /></beat>
+          <beat><chord root="D" /></beat>
+          <beat><chord root="A" /></beat>
+          <beat><chord root="G" /></beat>
+        </lines>
+      </instrument>
+    </lyrics>
+
+<instrumental> tags are siblings to <verse> tags. They can be in any order
+(described in ``<verseOrder>``). The name of an instrumental part can be intro (``name="i"``),
+middle (``name="m"``), outro (``name="o"``) or solo (``name="s"``), and can named similar to other
+verse names (``i, i1, i2, i1a, i1b``). Instrumental part can't contain lyrics, only ``<chord>`` and
+``<beat>`` tags. A ``<beat>`` represents a beat in the music. It is not
+mandatory to separate beats, instrumental parts can contain only chords.
```

Comment:
Could you explain more how the syntax with 'beat' tag should look like?

Reply:
Will it be good?

A ``<beat>`` represents a beat in the music. A ``<beat>`` tag can contains only ``<chord>`` tags. 
But it is not mandatory to separate beats, instrumental parts can contain chords only::

```xml
    <instrument name="i">
      <lines>
        <chord root="D" /><chord root="A" /><chord root="G" />
      </lines>
    </instrument>
```

Reply:
Yes, that is ok.

Reply:
Thanks. Done.